tests/test_coefficient.py

```diff
@@ -17,10 +15,10 @@ def test_coefficient_proratisation_only_contract_periods_wide():
         contrat_de_travail_fin='2017-12-01',
         allegement_fillon_mode_recouvrement=u'progressif'))
     simulation = scenario.new_simulation()
-    assert_equal(simulation.calculate('coefficient_proratisation', '2017-11'), 1)
-    assert_equal(simulation.calculate('coefficient_proratisation', '2017-12'), 0)
-    assert_equal(simulation.calculate('coefficient_proratisation', '2017-10'), 0)
-    assert_equal(simulation.calculate_add('coefficient_proratisation', '2017'), 1)
+    assert simulation.calculate('coefficient_proratisation', '2017-11') == 1
+    assert simulation.calculate('coefficient_proratisation', '2017-12') == 0
+    assert simulation.calculate('coefficient_proratisation', '2017-10') == 0
+    assert simulation.calculate_add('coefficient_proratisation', '2017') == 1
```

Comment:
> A little puzzled about why we're doing this. Error messages printed by nose helpers are _much_ more readable and informative than the basic `AssertionError` printed by a bare `assert`. Did we ever encountered issues with nose?

Reply:
To illustrate my point:

With:

```py
from nose.tools import assert_equal

x = [1,2]
y = [1,2,3]
```

If we do `assert x == y`
We get:
```
  File "test.py", line 6, in <module>
    assert x == y
AssertionError
```

While with `assert_equal(x,y)`
We get:
```
AssertionError: Lists differ: [1, 2] != [1, 2, 3]

Second list contains 1 additional elements.
First extra element 2:
3

- [1, 2]
+ [1, 2, 3]
?      +++
```

This seems like a _big_ loss of information and implies _a lot of work_ to understand what happen when a test fail.

Reply:
See https://docs.pytest.org/en/latest/assert.html

Reply:
Wow, I didn't try with Pytest, and indeed the message is great. This is a nice feature!

```
    def test_pytest():
    
>     assert x == y
E     assert [1, 2] == [1, 2, 3]
E       Right contains more items, first extra item: 3
E       Full diff:
E       - [1, 2]
E       + [1, 2, 3]
E       ?      +++
```

Thanks for the reference!